docs/moduleAna5Utils.rst

```diff
@@ -0,0 +1,102 @@
+
+#####################################################################
+ana5Utils
+#####################################################################
+
+
+distanceTimeAnalysis: Visualizing the temporal evolution of flow variables
+------------------------------------------------------------------------------
+
+With the functions gathered in this module, flow variables of avalanche simulation results can be
+visualized in a distance versus time diagram, the so called **thalweg-time diagram**.
+The **tt-diagram** provides a new way to identify main features of the temporal evolution of
+a flow variables along the :term:`avalanche path`.
+This is based on the ideas presented in :cite:`FiFrGaSo2013` and :cite:`RaKo2020`, where
+avalanche simulation results have been transformed into the radar coordinate system to facilitate
+direct comparison, combined with the attempt to analyze simulation results in an avalanche path
+dependent coordinate system (:cite:`Fi2013`).
+In addition to the **tt-diagram**, :py:mod:`ana5RangeTime` also offers the possibility to
+produce simulated **range-time diagrams** of the flow variables with respect to a radar's field
+of view. In this manner, simulation results can be directly compared to radar measurements in terms
+of front position and inferred approach velocity. The colorcoding of the simulated
+**range-time** diagrams refers to average values of the chosen flow parameter
+(e.g. flow depth, flow velocity) at specified range gates. This colorcoding is not directly
+comparable to the intensity given in the range-time diagram from radar measurements.
+
+.. Note::
+  The **tt-diagram** and the **range-time diagram** can be produced *during run time* of
+  :py:mod:`com1DFA`, or as a postprocessing step. However, the second option requires saving all the
+  required time steps of the flow parameter fields, which is much more computationally expensive
+  compared to the first option.
+
```

Comment:
Even if it is only a post-processing step, you would need to alter the output to write each timestep. I disagree because it produces so much overload and is only feasible for a couple of simulations.

Reply:
That is true for the generation of the data, but not for the plotting itself. Similar to saving particles, but doing the postprocessing/visualization i.e. with paraview...
I.e preparation/aggregation of data is fine... @awirb will change it as discussed in meeting.

Reply:
now we save the mtiInfo as pickle and this is available for plotting afterwards - so no need to save all the result fields but plotting is not inside com1DFA